recipe/meta.yaml

```diff
@@ -7,110 +7,50 @@ package:
   build:
     - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
     - libtool
     - make
     - patch
   host:
     - pkg-config
+    # Need a separate host section so that ncurses doesn't remove -L$PREFIX/lib
+    - libcxx        # [osx]
+    - libstdcxx-ng  # [linux]
 
 test:
 # TODO :: conda-build needs seperate test/requires_build and test/requires_host
-#  requires:
-#    - pkg-config
-  commands:
-    # Test libraries
-    {% set ncurses_libraries = [
-        "libncurses",
-        "libtinfo",
-        "libform",
-        "libmenu",
-        "libpanel",
-    ] %}
-    {% for each_ncurses_library in ncurses_libraries %}
-    - test -f ${PREFIX}/lib/{{ each_ncurses_library }}.a
-    - test -f ${PREFIX}/lib/{{ each_ncurses_library }}w.a
-    - test -f ${PREFIX}/lib/{{ each_ncurses_library }}.dylib    # [osx]
-    - test -f ${PREFIX}/lib/{{ each_ncurses_library }}w.dylib   # [osx]
-    - test -f ${PREFIX}/lib/{{ each_ncurses_library }}.so       # [linux]
-    - test -f ${PREFIX}/lib/{{ each_ncurses_library }}w.so      # [linux]
-    {% endfor %}
-
-    # Test include directories
-    - test -d ${PREFIX}/include/ncurses
-    - test -d ${PREFIX}/include/ncursesw
-
-    # Test headers
-    {% set ncurses_headers = [
-        "curses.h",
-        "cursesapp.h",
-        "cursesf.h",
-        "cursesm.h",
-        "cursesp.h",
-        "cursesw.h",
-        "cursslk.h",
-        "eti.h",
-        "etip.h",
-        "form.h",
-        "menu.h",
-        "nc_tparm.h",
-        "ncurses.h",
-        "ncurses_dll.h",
-        "panel.h",
-        "term.h",
-        "term_entry.h",
-        "termcap.h",
-        "tic.h",
-        "unctrl.h"
-    ] %}
-    {% for each_ncurses_header in ncurses_headers %}
-    - test -L ${PREFIX}/include/ncurses/{{ each_ncurses_header }}
-    - test -L ${PREFIX}/include/ncursesw/{{ each_ncurses_header }}
-    - test -f ${PREFIX}/include/{{ each_ncurses_header }}
-    {% endfor %}
-
-    # Test pkg-config files
-    {% set ncurses_pc_files = [
-        "form",
-        "menu",
-        "ncurses++",
-        "ncurses",
-        "panel",
-        "tinfo"
-    ] %}
-    {% for each_ncurses_pc_file in ncurses_pc_files %}
-    - test -f ${PREFIX}/lib/pkgconfig/{{ each_ncurses_pc_file }}.pc
-    - test -f ${PREFIX}/lib/pkgconfig/{{ each_ncurses_pc_file }}w.pc
-    {% endfor %}
-
-    # Test ncurses library arguments.
-    # - pkg-config ncursesw --libs | grep "^-L${PREFIX}/lib -lncursesw -ltinfow$"
-    # - pkg-config tinfow --libs | grep "^-L${PREFIX}/lib -ltinfow$"
-    # - pkg-config ncurses --libs | grep "^-L${PREFIX}/lib -lncurses -ltinfo$"
-    # - pkg-config tinfo --libs | grep "^-L${PREFIX}/lib -ltinfo$"
+  requires:
+    - pkg-config
 
 about:
-  home: http://www.gnu.org/software/ncurses/
-  license: Free software (MIT-like)
+  home: https://invisible-island.net/ncurses/
+  # See https://github.com/mirror/ncurses/blob/3ef920d65fb2d4046096131d868ae8d4bba79d46/package/debian/copyright
+  license: X11 AND BSD-3-Clause
```

Comment:
I think license should be MIT. As in https://github.com/ThomasDickey/ncurses-snapshots/blob/v6_4/COPYING

The debian package seems to be used to test ncurses builds:
https://github.com/mirror/ncurses/blob/3ef920d65fb2d4046096131d868ae8d4bba79d46/package/debian/copyright#L2

Reply:
I would say `MIT and X11`. Is it correct?

Reply:
yes, MIT and X11 seems to be the correct choice